test/utf8-characters.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# This logging test ensures that a log message containing
+# extended UTF-8 characters will be correctly processed by
+# Fluentd and appear in Elasticsearch.
+
+source "$(dirname "${BASH_SOURCE[0]}" )/../hack/lib/init.sh"
+source "${OS_O_A_L_DIR}/deployer/scripts/util.sh"
```

Comment:
util.sh has been moved to ${OS_O_A_L_DIR}/hack/testing in the latest master and release-3.6 branch.

Reply:
Good catch. Fixed.